resources/certificate.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This file should be called `self_signed_certificate_for_testing.crt` to make its purpose clear ;)